example/src/main/java/com/genius/groupie/example/MainActivity.java

```diff
@@ -70,7 +70,7 @@ protected void onCreate(Bundle savedInstanceState) {
         gray = ContextCompat.getColor(this, R.color.background);
         betweenPadding = getResources().getDimensionPixelSize(R.dimen.padding_small);
 
-        groupAdapter = new GroupAdapter(this);
+        groupAdapter = new GroupAdapter();
```

Comment:
I still have nightmares because of some of the bugs caused by this ItemClickListener :D  , it was so bad !  
If you think we should keep it though, yeah just having it as a setter sounds like a good compromise.